microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -63,8 +63,13 @@ See the following list for details:
 
 [id="microshift-4-16-custom-cert-auths"]
 ==== Customizable certificate authorities for the API server are supported
-With this release, you can configure a custom server certificate that has been issued by an external certificate authority (CA). The default API server certificate is issued by an internal {microshift-short} cluster CA. You can now replace this certificate with one that is issued by a CA that clients trust.
-//TODO add xref after section merges
+With this release, you can configure a custom server certificate that has been issued by an external certificate authority (CA). The default API server certificate is issued by an internal {microshift-short} cluster CA. You can now replace this certificate with one that is issued by a CA that clients trust. See xref:../microshift_configuring/microshift-custom-ca.adoc#microshift-custom-ca[Configuring custom certificate authorities] for details.
+
+[id="microshift-4-16-audit-logging-config"]
+==== Configurable policies for log file rotation and retention
+
+You can now configure audit logging policies to manage the disk usage for log files, ensuring that edge devices with limited storage capacities are not hampered by logging data accumulation. To configure audit log policies, use settings such as a maximum storage limit for the events that you want logged. You can also chose a profile that determines system behavior when these maximums are reached. Using these settings gives you control over audit log rotation, retention, and maximum allowable storage.
```

Comment:
The audit logging work does not provide a direct means of enforcing a storage limit for data.  Instead, it provides access to existing auditing settings for max file age, max file size, and total number of files retained.  Policy profiles specify different degrees of logging granularity.  I'd suggest a change to something like:

```suggestion
You can now configure audit logging policies to manage the retention policies for log files, ensuring that edge devices with limited storage capacities are not hampered by logging data accumulation. To configure audit log policies, use settings such as a maximum file size limit and maximum retained files to set an upper limit on log storage size. You can also choose an audit policy profile to specify the granularity of data collected. Using these settings gives you control over audit log rotation, retention, and, indirectly, maximum allowable storage.
```

Reply:
Tweaked a bit and removed the last sentence.